drivers/gpio/gpio_mcux_lpc.c

```diff
@@ -61,7 +63,7 @@ struct gpio_mcux_lpc_data {
 	/* pin association with PINT id */
 	pint_pin_int_t pint_id[32];
 	/* ISR allocated in device tree to this port */
-	uint32_t isr_list[8];
+	uint32_t isr_list[MAX_IRQ];
```

Comment:
This is being change as part of this PR https://github.com/zephyrproject-rtos/zephyr/pull/37148/commits/db645c0c83109945a0304884e6c2ebd11569a837